src/tools/run-make-support/src/lib.rs

```diff
@@ -149,6 +149,50 @@ pub fn bin_name(name: &str) -> String {
     if is_windows() { format!("{name}.exe") } else { name.to_string() }
 }
 
+fn ar<P: AsRef<str>, P2: AsRef<str>>(obj_path: P, lib_path: P2, caller_line_number: u32) {
+    let mut ar = Command::new(env::var("AR").unwrap());
+    ar.current_dir(tmp_dir()).arg("crus").arg(lib_path.as_ref()).arg(obj_path.as_ref());
+    let output = ar.output().unwrap();
+    if !output.status.success() {
+        handle_failed_output(&ar, output, caller_line_number);
+    }
+}
+
+/// Returns the path for the given library name.
```

Comment:
Question: this helper doesn't actually return the path?

Reply:
Indeed, completely changed what I planned to do and forgot to update the comment.